sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -117,7 +117,7 @@ statement
         tableIdentifier partitionSpec?                                 #loadData
     | TRUNCATE TABLE tableIdentifier partitionSpec?
         (COLUMNS identifierList)?                                      #truncateTable
-    | ADD identifier .*?                                               #addResource
+    | op=(ADD | DELETE | LIST) identifier .*?                          #manageResource
```

Comment:
Seems that `DELETE` isn't implemented in this PR. Maybe remove changes related to `DELETE` and do it in another PR?